python/pypto/ir/op/distributed/tensor_ops.py

```
@@ -156,4 +156,28 @@ def get(
+    result type is ``target``'s :class:`ir.DistributedTensorType` (the rebind
+    target — same semantics as :func:`pl.store`).
+
+    LowerCompositeOps expands this into the 4-phase
+    notify/wait/remote_load+accumulate/store decomposition; this Call never
+    survives past that pass.
+    """
+    actual_span = _get_span_or_capture(span, frame_offset=1)
+    return _ir_core.create_op_call("pld.tensor.allreduce", [target, signal], {"op": int(op)}, actual_span)
+
+
+__all__ = ["allreduce", "alloc_window_buffer", "get", "put", "window"]
```

Ruff (0.15.15)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)